.kitchen.yml

```diff
@@ -56,3 +59,19 @@ suites:
   attributes:
     varnish:
       version: '4.0'
+      log_daemon: false
+      
+- name: distro_libraries
```

Comment:
Ah okay. Calling it `package` and `vendor` might be more intuitive. What do others think? I'm torn, but I distro wasn't immediately obvious to me. Maybe it's just cuz I'm varnish ignorant.